apis/kubernetes/v1alpha1/doc.go

```diff
@@ -14,9 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha2
-
-// Hub marks this type as a conversion hub.
-func (g *Object) Hub() {
-
-}
+// Package v1alpha1 contains the v1alpha1 group Object resources of the Kubernetes provider.
+// +kubebuilder:object:generate=true
+// +groupName=kubernetes.crossplane.io
+// +versionName=v1alpha1
+package v1alpha1
```

Comment:
why are we moving back from v1alpha2 to v1alpha2?

Reply:
SPKS is not yet on the latest provider-kubernetes

Reply:
we are not. we are just adding it temporarily as SPKS still uses the old provider-kubernetes